backend/EasyMeets.Core/EasyMeets.Core.BLL/MappingProfiles/MeetingProfile.cs

```diff
@@ -56,14 +52,26 @@ private List<UserMeetingDTO> GetThreeMembersForMeeting(Meeting meeting)
         private IEnumerable<UserMeetingDTO> GetAllParticipants(Meeting meeting)
         {
             var slotMembers = meeting.MeetingMembers
-                .Select(x => new UserMeetingDTO 
-                    { Name = x.TeamMember.User.Name, Email = x.TeamMember.User.Email, TimeZone = x.TeamMember.User.TimeZone.ToString() }).ToList();
+                .Select(x => new UserMeetingDTO
+                {
+                    Name = x.TeamMember.User.Name,
+                    Email = x.TeamMember.User.Email,
+                    TimeZoneName = x.TeamMember.User.TimeZoneName,
+                    TimeZoneValue = x.TeamMember.User.TimeZoneValue,
+                })
+                .ToList();
 
             if (meeting.AvailabilitySlot is not null)
             {
                 var external = meeting.AvailabilitySlot.ExternalAttendees
                     .Select(x => new UserMeetingDTO
-                        { Name = x.Name, Email = x.Email, TimeZone = x.TimeZone.ToString() }).ToList();
+                    {
+                        Name = x.Name,
+                        Email = x.Email,
+                        TimeZoneName = x.TimeZoneName,
+                        TimeZoneValue = x.TimeZoneValue,
+                    })
```

Comment:
use mapper

Reply:
this code is using in mapper profile. How we can use mapper in his own profile ?

Reply:
Maybe we can map class to class in mapper, but select logic and other queries place outside?

Reply:
https://prnt.sc/jFFto1PdGC9q - problem with this strange mapping. I don't know who write this, but I think I should correct ...